cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -54,7 +54,7 @@ const (
 )
 
 // NewCloudControllerManagerCommand creates a *cobra.Command object with default parameters
-func NewCloudControllerManagerCommand() *cobra.Command {
+func NewCloudControllerManagerCommand(newControllerInitializers map[string]InitFunc, cloud cloudprovider.Interface) *cobra.Command {
```

Comment:
s/newControllerInitializers/controllerInitializers

Reply:
Thanks. Addressed in the latest changes.